### ISSUE TITLE
Upgrade vonage/client

### ISSUE THREAD

**Issue by martio** [Nexmo/nexmo-laravel#68]
Please, upgrade `vonage/client` to version 3.0.0

For PHP 8.1:

```
Return type of Vonage\Client\Credentials\AbstractCredentials::offsetGet($offset) should either be compatible with ArrayAccess::offsetGet(mixed $offset): mixed, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in /var/www/html/vendor/vonage/client-core/src/Client/Credentials/AbstractCredentials.php on line 31
```

**Comment by SecondeJK** [comment]
This is underway

**Comment by SecondeJK** [comment]
Fixed with #67 , release published. Thank you @martio !
